src/main/java/cz/xtf/TestConfiguration.java

```diff
@@ -109,510 +35,231 @@
 	public static final String IMAGE_H2 = "xtf.h2";
 	public static final String IMAGE_MSA = "xtf.msa";
 	public static final String IMAGE_ZIPKIN = "io.zipkin.java";
-	public static final String IMAGE_RAD = "xtf.rad.spark";
-	public static final String IMAGE_OSHINKO_REST = "xtf.rad.oshinko.rest";
-	public static final String IMAGE_OSHINKO_WEBUI = "xtf.rad.oshinko.webui";
-	public static final String IMAGE_OSHINKO_CLI = "xtf.rad.oshinko.cli";
-	public static final String IMAGE_PY_SPARK = "xtf.rad.driver.python";
-	public static final String IMAGE_JAVA_SPARK = "xtf.rad.driver.java";
-	public static final String IMAGE_SCALA_SPARK = "xtf.rad.driver.scala";
 	public static final String IMAGE_SQUID = "org.squid-cache";
 	public static final String IMAGE_TCP_PROXY = "xtf.tcp-proxy";
 	public static final String IMAGE_MM_SERVICE = "org.hawkular.service";
 	public static final String IMAGE_MM_DATASTORE = "org.hawkular.datastore";
-
 	public static final String VERSION_EAP = "xtf.version.eap";
 	public static final String VERSION_JDV = "xtf.version.jdv";
 	public static final String VERSION_JDG = "xtf.version.jdg";
 	public static final String VERSION_EWS = "xtf.version.ews";
-	public static final String VERSION_FUSE = "xtf.version.fuse";
 	public static final String VERSION_KIE = "xtf.version.kie";
 	public static final String VERSION_JDK = "xtf.version.jdk";
 	public static final String VERSION_SSO = "xtf.version.sso";
 	public static final String VERSION_AMQ = "xtf.version.amq";
 	public static final String VERSION_MSA = "xtf.version.msa";
-
-	public static final String CDK_DOMAIN = "cd.xtf.cz";
-	public static final String CDK_IP = "10.1.2.2";
+	public static final String VERSION_FUSE = "xtf.version.fuse";
 	public static final String CDK_INTERNAL_HOSTNAME = "localhost.localdomain";
-
-	public static final String OC_BINARY_LOCATION = "oc.binary.location";
-
 	public static final String CI_USERNAME = "ci.username";
 	public static final String CI_PASSWORD = "ci.password";
 
-	private static final Logger LOGGER = LoggerFactory.getLogger(TestConfiguration.class);
+	private Properties properties;
 	private static final TestConfiguration INSTANCE = new TestConfiguration();
```

Comment:
Same as above. No need to hold INSTANCE. All the properties are held in XTFConfiguration.